src/node.cc

```diff
@@ -3947,6 +3953,8 @@ static void ParseArgs(int* argc,
       Revert(cve);
     } else if (strcmp(arg, "--preserve-symlinks") == 0) {
       config_preserve_symlinks = true;
+    } else if (strcmp(arg, "--harmony-modules") == 0) {
```

Comment:
Is there a batter flag name we can use?

Aside: `--experimental=feature,s` seems like it may be a more desirable path considering a multiple-features interaction.

Reply:
this is somewhat a legacy since v8's options used to call it this. fine renaming

Reply:
I realize that but I'm not sure that naming is useful anymore?

`--experimental-modules` might be better?

Reply:
changed